compiler/quilt/test/test_build.py

```diff
@@ -49,7 +49,7 @@ def test_build_from_cache(self):
 
         # Verify cache contents
         srcpath = os.path.join(mydir, 'data/10KRows13Cols.csv')
-        path_hash = build._path_hash(srcpath, 'csv', {})
+        path_hash = build._path_hash(srcpath, 'csv',  {'parse_dates': ['Date0']})
```

Comment:
this works but it's super fragile; @kevinemoore can you get this info from build.yml? other unit tests will need the logic.